src/main/java/mate/academy/WebDtoApplication.java

```diff
@@ -0,0 +1,12 @@
+package mate.academy;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class WebDtoApplication {
```

Comment:
```suggestion
public class BookStoreApplication {
```
unusual name for the main class